optax/_src/loss.py

```diff
@@ -358,3 +358,26 @@ def loop_body(prev, x):
   per_seq_loss = -jnp.einsum('bn,bn->b', logalpha_phi_last, one_hot)
 
   return per_seq_loss
+
+
+def kldiv_loss(predictions: chex.Array,
+               targets: chex.Array):
+  """Computes the Kullback-Leibler divergence (relative entropy) loss.
+
+  Measures the information gain achieved if probability distribution P
```

Comment:
Nit: since you don't refer to P and Q elsewhere I don't think it's necessary to introduce these names (you could write it in terms of a different distribution and a reference distribution). But as I said, this is a nit - it's perfectly clear what you mean.